addons/website_sale/static/src/snippets/s_dynamic_snippet_products/000.scss

```diff
@@ -16,4 +56,19 @@
             --o-wsale-card-border-width: 1px;
         }
     }
+
+    // Applies arrows' alignment except when title is on the left + desktop
+    &:where(.o_wsale_products_opt_design_thumbs) {
+        &:not(:has(.s_dynamic_snippet_title_aside)) {
+            @include carouselControls;
+            @include colSizeRules;
```

Comment:
Rather than using a mixin, I'd suggest to use a placeholder selector instead, looks more appropriate.
Mixins are indeed supposed to "do stuff", while [placeholder selector](https://sass-lang.com/documentation/style-rules/placeholder-selectors/) are a great way to share code between selectors and avoid code duplication.

note: my previous comment about scoping mixins, is valid for placeholder selectors as well ;)